src/content/structured/accessibility/documenting/accessibility-statements-guidance.mdx

```diff
@@ -14,7 +14,7 @@ contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/struc
 
 import { IcAlert } from "@ukic/react";
 
-## Introduction
+## How to fill in an accessibility statement
```

Comment:
I wonder if this should be capitalised - looks like it is pretty much everywhere else, including the paragraph below
```suggestion
## How to fill in an Accessibility Statement
```